Cargo.toml

```diff
@@ -56,7 +56,7 @@ with-runtime = ["dep:cairo-native-runtime"]
 normal = ["aquamarine"]
 
 [dependencies]
-aquamarine = "0.5.0"
+aquamarine = "0.6.0"
```

Comment:
I just noticed, but shouldn't this be in `dev-dependencies`?

Reply:
we can move it in another pr